Add Committer role to the project (#79)
PrestaShop/open-source: content/maintainers-guide/reviewing-pull-requests.md

## Approving Pull Requests

> Maintainers and reviewers are responsible for the Pull Requests they approve. The main objective of this process is to make sure that the change being reviewed does not produce undesired side effects and that it's inclusion is coherent with the project's objectives.

Reviewer: ```suggestion
Reviewers with merge rights (Committers and Maintainers) are responsible for the Pull Requests they approve. The main objective of this process is to make sure that the change being reviewed does not produce undesired side effects and that its inclusion is coherent with the project's objectives.
```

### Discussing with author

> During review, maintainers or reviewers can ask questions to the Pull Request author (for example to better understand a technical choice). Simply writing the question as a comment is enough.

Reviewer: ```suggestion
Reviewers might ask questions to the Pull Request author (for example to better understand a technical choice). Simply writing the question as a comment is enough.
```

> When doing so, it is recommended to add the label "Waiting for author" on the Pull Request. This helps other maintainers or reviewers to know this Pull Request state.

Reviewer: ```suggestion
When doing so, it is recommended to add the label "Waiting for author" on the Pull Request. This helps other reviewers to know the state of this Pull Request.
```

A Pull Request may only be merged after the following requirements have been fulfilled:

- The change must not have any outstanding merge conflicts.

> - The change has been approved by at least one maintainer

Reviewer: ```suggestion
- The change has been approved by at least one maintainer.
```

- Automated checks (including automated tests) are passing.
- The change has been approved by the QA team using the "QA ✓" label, unless there's a general agreement that the change is not testable by QA.

> On the [PrestaShop Core repository][prestashop-core-repository]), two approvals are required:
> - either two approvals set by two maintainers
> - or one approval by a reviewer and one approval by a reviewer

Reviewer: ```suggestion
On the [PrestaShop Core repository][prestashop-core-repository]), Pull Requests must be approved by two reviewers with merge rights, including at least one maintainer. For example:

- One committer and one maintainer
- Two maintainers
```

A Pull Request may only be merged after the following requirements have been fulfilled:

> - The change has received all the required approvals

Reviewer 2: ```suggestion
- The change has received all the required approvals.
```
